frontend/src/entry/settings/Settings.tsx

```diff
@@ -102,6 +108,17 @@ export default function Settings({ entry }: SettingsProps) {
             >
               Template
             </SimpleListItemButton>
+            {!entry.settings.mirror?.sourceModelId &&
```

Comment:
If we handle the uiConfig error and loading states as I've mentioned in another comment this can be simplified:
```
            {!entry.settings.mirror?.sourceModelId &&
              uiConfig.modelMirror.enabled && (
                <SimpleListItemButton
                  selected={selectedCategory === SettingsCategory.MIRRORED_MODELS}
                  onClick={() => handleListItemClick(SettingsCategory.MIRRORED_MODELS)}
                >
                  Mirrored Models
                </SimpleListItemButton>
              )}
```